README.md

```diff
@@ -200,18 +383,26 @@ memcached-for-wordpress-65b75fd8c9-7b9x7   1/1       Running   0          14m
 
 ## Conclusion
 
-Hopefully, this guide was an effective demonstration of the value of the Operator Framework for building and managing Operators. There is much more that we left out in the interest of brevity. The Operator Framework and its components are open source, so please feel encouraged to jump into each individually and learn what else you can do. If you want to discuss your experience, have questions, or want to get involved, join the Operator Framework mailing list [here][mailing_list].
+Hopefully, this guide was an effective demonstration of the value of the Operator Framework for building and managing operators. There is much more that we left out in the interest of brevity. The Operator Framework and its components are open source, so please feel encouraged to jump into each individually and learn what else you can do. If you want to discuss your experience, have questions, or want to get involved, join the Operator Framework [mailing list][mailing_list].
 
 <!---  Reference URLs begin here -->
 
-[blog_post]: https://coreos.com/blog/introducing-operator-framework
+[org_operator_framework]: https://github.com/operator-framework/
+[site_blog_post]: https://coreos.com/blog/introducing-operator-framework
 [operator_sdk]: https://github.com/operator-framework/operator-sdk
 [operator_lifecycle_manager]: https://github.com/operator-framework/operator-lifecycle-manager
+[site_memcached]: https://memcached.org/
+[doc_sdk_user_guide]: https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md
+[doc_sdk_install_instr]: https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#install-the-operator-sdk-cli
+[doc_olm_install_instr]: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md
 [layout_doc]: https://github.com/operator-framework/operator-sdk/blob/master/doc/project_layout.md
-[quay_io]: https://www.quay.io
-[kubernetes_cr]: https://kubernetes.io/docs/concepts/api-extension/custom-resources/
-[handler_go]: https://github.com/operator-framework/getting-started/blob/master/handler.go.tmpl#L7
-[create_public_image]: https://quay.io/new/
+[manager_go_doc]: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/manager#Manager
+[controller_go_doc]: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg#hdr-Controller
+[memcached_controller]: https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/pkg/controller/memcached/memcached_controller.go
```

Comment:
We should point to the example controller used specifically for the user-guide.
https://github.com/operator-framework/operator-sdk/blob/master/example/memcached-operator/memcached_controller.go.tmpl

The samples repo memcached-operator has a few differences like the import path being different from what we use in the example.